apps/gitness/src/pages/pull-request-commits-page.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import {
+  ListPagination,
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+  Spacer
+} from '@harnessio/canary'
+import { NoData, PullRequestCommits, SkeletonList } from '@harnessio/playground'
+import { useListPullReqCommitsQuery, TypesCommit } from '@harnessio/code-service-client'
+
+export default function PullRequestCommitsPage() {
+  const { data: commitData, isFetching } = useListPullReqCommitsQuery(
+    {
+      repo_ref: 'workspace/repo/+',
+      pullreq_number: 1,
+      queryParams: { page: 0, limit: 10 }
+    },
+    /* To enable mock data */
+    {
```

Comment:
JFYI adding `placeholderData` data is optional while using a hook to make api call.

Reply:
got it